evap/static/css/evap.scss

```diff
@@ -1259,3 +1259,65 @@ table.dataTable thead>tr>th.sorting_asc, table.dataTable thead>tr>th.sorting_des
         flex: 0 0 40%;
     }
 }
+
+
+/* interactive slider */
```

Comment:
`interactive text answer review slider` should make it easier to imagine the respective element